pkg/install/1-installresources.go

```diff
@@ -130,21 +130,15 @@ func (i *Installer) installResources(ctx context.Context) error {
 							TTL: to.Int64Ptr(300),
 							ARecords: &[]mgmtprivatedns.ARecord{
 								{
-									Ipv4Address: to.StringPtr(
-										fmt.Sprintf(
-											"[reference('Microsoft.Network/loadBalancers/aro-internal-lb', '%s')"+
-												".frontendIpConfigurations[0].properties.privateIPAddress]",
-											apiVersions["network"],
-										),
-									),
+									Ipv4Address: to.StringPtr(fmt.Sprintf("[reference('Microsoft.Network/loadBalancers/aro-internal-lb', '%s').frontendIpConfigurations[0].properties.privateIPAddress]", apiVersions["network"])),
 								},
 							},
 						},
 					},
 					APIVersion: apiVersions["privatedns"],
 					DependsOn: []string{
-						"Microsoft.Network/privateDnsZones/" + installConfig.Config.ObjectMeta.Name + "." + installConfig.Config.BaseDomain,
 						"Microsoft.Network/loadBalancers/aro-internal-lb",
+						"Microsoft.Network/privateDnsZones/" + installConfig.Config.ObjectMeta.Name + "." + installConfig.Config.BaseDomain,
```

Comment:
@jim-minter I see that the `DependsOn` slice has been reordered. There is another occurrence of the same slice which was not reordered in the same way (https://github.com/nilsanderselde/ARO-RP/blob/2488c3d5010b4812791c45c279d3ec209c71de3b/pkg/install/1-installresources.go#L159). Should that be changed as well?

Reply:
PR created: https://github.com/Azure/ARO-RP/pull/165